### PR TITLE
GUI First Stage Setup Hindi Translation

### PR DESCRIPTION
**Pull request by arnavbhatt288**
## Purpose
Creating hi-IN.rc and doing translations on it for first stage setup.

## Proposed changes
- Created hi-IN.rc and done translations on it
- Defined hi-IN.rc on reactos.rc

### CODE REVIEW

File: base/setup/reactos/lang/hi-IN.rc [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    IDS_DRIVESUBTITLE "इन्स्टलेशन विभाजन, सिस्टम फ़ोल्डर और बूट लोडर तैयार करें।"
+    IDS_PROCESSTITLE "सेटअप विभाजन, कॉपी फ़ाइलें और सेटअप सिस्टम"
+    IDS_PROCESSSUBTITLE "विभाजन बनाएं और फ़ॉर्मेट करें, फ़ाइलें कॉपी करें, बूट लोडर को इंस्टॉल करें और सेटअप करें"
+    IDS_RESTARTTITLE "सेटअप का पहला चरण समाप्त हो गया"
+    IDS_RESTARTSUBTITLE "सेटअप का पहला चरण पूरा हो गया है, दूसरे चरण के साथ जारी रखने के लिए रीस्टॉर्ट करें"
+    IDS_SUMMARYTITLE "इन्स्टलेशन सारांश"
+    IDS_SUMMARYSUBTITLE "इन्स्टलेशन डिवाइस पर लागू करने से पहले जांच करने के लिए इन्स्टलेशन गुणों की सूची"
+    IDS_ABORTSETUP "रिऐक्ट ओएस आपके कंप्यूटर पर पूरी तरह से इंस्टॉल नहीं है। अगर आप अभी सेटअप को छोड़ देते हैं, तो आपको रिऐक्ट ओएस इंस्टॉल करने के लिए सेटअप को फिर से चलाने की आवश्यकता होगी। आपको वास्तव में छोड़ना है?"
+    IDS_ABORTSETUP2 "इन्स्टलेशन रद्द करें?"
+    IDS_NO_TXTSETUP_SIF "'txtsetup.sif' खोजने में असमर्थ \nसेटअप जारी रखने में असमर्थ है ""
```
Comment (learn-more) [inline base/setup/reactos/lang/hi-IN.rc]:
```suggestion
    IDS_NO_TXTSETUP_SIF "'txtsetup.sif' खोजने में असमर्थ \nसेटअप जारी रखने में असमर्थ है "
```

**Comment by learn-more** [comment]
@HBelusca any update on this?

**Comment by HBelusca** [comment]
@learn-more : No, I haven't checked yet whether the translation contained the stuff that has been added to the first-stage since october.

**Comment by arnavbhatt288** [comment]
I am still alive, except I was working for Haiku.
I will continue translating ReactOS and I hope that I will make new PR soon :)

**Pull request merged**